blank-canvas-blocks/experimental-theme.json

```diff
@@ -161,6 +175,13 @@
 					"thickness": "2px",
 					"width": "150px"
 				},
+				"search": {
+					"button": {
+						"lineHeight": "27px",
+						"padding": "calc( .9 * var(--wp--custom--padding--horizontal) )",
```

Comment:
If we're making the padding on this button customizable by theme.json, should we do the same for the button block?

Reply:
Padding customizations are out (in this PR).  Doing so at the button block level makes problems and I believe we should avoid.  The cleanest way to express that padding opinion seems to be line-height.